UnityProject/Assets/Scripts/Core/Chat/ChatUI.cs

```diff
@@ -84,6 +84,11 @@ public class ChatUI : SingletonManager<ChatUI>
 		/// </summary>
 		public event System.Action OnChatWindowClosed;
 
+		[BoxGroup("Animation")] public float ChatFadeSpeed = 2f;
```

Comment:
Is this a 2 seconds animation?

Reply:
no, the speed is a weight of the lerp, not the speed of the animation

Reply:
How does it look?

Reply:
https://user-images.githubusercontent.com/34368774/174357213-1159f765-0a3b-460a-82e3-ad51ee6c133b.mp4

Reply:
personally, I  think the fade-in is too slow for a default value, might just be me

Reply:
Might to get annoying with the mouseover if you're holding a gun?

Reply:
Can you make it last half the time it is taking in the video?